client/landing/stepper/hooks/use-send-email-verification.ts

```diff
@@ -1,7 +1,14 @@
 import wpcom from 'calypso/lib/wp';
 
-export function useSendEmailVerification() {
+export interface ResendEmailVerificationBody {
+	from: string; // Helps decide the url of the page after confirmation.
+}
+
+export function useSendEmailVerification( body: ResendEmailVerificationBody ) {
 	return async () => {
-		return wpcom.req.post( '/me/send-verification-email', { apiVersion: '1.1' } );
+		return wpcom.req.post( '/me/send-verification-email', {
+			apiVersion: '1.1',
+			...body,
```

Comment:
do we need to provide a default or fallback for when body does not exist?

Reply:
Added `{}` as default value.